solr/solrconfig.xml

```diff
@@ -0,0 +1,1905 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Is this file necessary, or would installing Solr provide a similar default file? If there aren't any reddit specific customizations in it, please omit it.